Client/CommandSerializer.cpp

```diff
@@ -1,9 +1,17 @@
 #include "CommandSerializer.h"
 
-constexpr unsigned char ESCAPED_BYTE_SENTRY = 61;
-constexpr unsigned char ESCAPED_60 = 44;
-constexpr unsigned char ESCAPED_61 = 45;
-constexpr unsigned char ESCAPED_62 = 46;
+/* 
+ * Because 
+ * 	0x3E represents beginning of packet
+ * 	0xeC represents end of packet
+ * we need to escape these in the packet payload
+*/
+constexpr unsigned char ESCAPED_BYTE_SENTRY = 61;   // 0x3D
+constexpr unsigned char ESCAPED_60 = 44;	    // 0x2C
+constexpr unsigned char ESCAPED_61 = 45;	    // 0x2D
+constexpr unsigned char ESCAPED_62 = 46;   	    // 0x2E
+
+
 constexpr int MAX_STEPS_WH_1000_XM3 = 19;
```

Comment:
There should probably also be MAX_STEPS_WH_1000_XM4 now, or this should be changed to a more generic name?

Reply:
I think that my WH-1000XM4 has 20 levels + wind noise reduction. So it should be 20 for 1000XM4. But I did not change it as ASM/NC worked quite as well as level 20 even at level 19. It could be changed, but that would mean handling it differently for different devices. I did no work on ASM/NC so I did not touch that part.